### DISCUSSION TITLE
Application for GSOC 2022 as a Student

### DISCUSSION THREAD

**Discussion by Albert-Sharma** [alan-turing-institute/sktime#2258] [General]
Greetings,
I am Sophomore at NIT-Hamirpur, and is still in learning process in the filed of Data Science. I want to sharpen my skills, by working for this years GSOC, but Don't have any proper guidance in-person to get a head start with it.
It will be my pleasure, if you could help me by showing me a path to start, though it may sound armature.

**Comment by fkiraly** [comment]
Great to hear of your interest, @Albert-Sharma!

I would advise you look at the GSoC 2022 application instructions:
https://github.com/sktime/mentoring
in there are also pointers on how to get started.

Please let us know if you have any questions!

**Reply by keerthanareddy98** [reply]
I'm a newbie to open source, and I want to participate in Gsoc(Google summer of code) this year 
I'm fascinated in python and would like to work with sktime and contribute to open source projects
Can you guide me through this process
Resources available for sktime and how do I execute the code on GitHub?
Thank you.

**Reply by fkiraly** [reply]
of course! See here:

- how to get started issue https://github.com/alan-turing-institute/sktime/issues/1147
- GSoC 2022 applications https://github.com/sktime/mentoring/blob/main/internships/gsoc2022.md
- 2022 project ideas https://github.com/sktime/mentoring/blob/main/internships/projects_2022.md

**Comment by Albert-Sharma** [comment]
Greetings,
Thank you very much, sir, for your advice, I am currently going through the
sources you mentioned.
Is there any particular social platform to have a healthy interaction with
the community?
Regards.

**Reply by fkiraly** [reply]
> Is there any particular social platform to have a healthy interaction with
the community?

Yes, the one mentioned in point 1 when you follow the 1st bullet point in my previous post.

**Comment by Albert-Sharma** [comment]
Thank you for your response.
